csharp/ql/src/experimental/Security Features/campaign/Solorigate/ModifedFnvFunctionDetection.qhelp

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE qhelp PUBLIC
```

Comment:
Looks like this file name contains a typo; it should probably be: `ModifiedFnvFunctionDetection.qhelp` (note the missing *i*)

Reply:
Fixing it